go.mod

```diff
@@ -11,9 +11,9 @@ require (
 	github.com/kubernetes-csi/csi-lib-utils v0.14.0
 	github.com/kubernetes-csi/csi-test/v4 v4.4.0
 	google.golang.org/grpc v1.57.0
-	k8s.io/api v0.27.4
-	k8s.io/apimachinery v0.27.4
-	k8s.io/client-go v0.27.4
+	k8s.io/api v0.28.0-rc.0
+	k8s.io/apimachinery v0.28.0-rc.0
+	k8s.io/client-go v0.28.0-rc.0
 	k8s.io/csi-translation-lib v0.27.4
```

Comment:
This one also has https://github.com/kubernetes/csi-translation-lib/tree/v0.28.0-rc.0 it should not be 0.27.4, something is wrong with the script.

Reply:
I tried running `go mod tidy` but even that added version `v0.27.4`. For [livenessprobe](https://github.com/kubernetes-csi/livenessprobe/pull/201#discussion_r1283719137) manually updating this to v0.28-rc.0 caused failure so did not try this here. I have pushed a commit now to update `csi-translation-lib` to  v0.28-rc.0, hoping it doesn't error here.